test/support/OffLineBuilder.js

```diff
@@ -0,0 +1,60 @@
+const sinon = require('sinon');
+const functionHelper = require('../../src/functionHelper');
+const Offline = require('../../src/index');
+const ServiceBuilder = require('./ServerlessBuilder');
+
+
+function createHandler(handlers) {
+    return (funOptions) => {return handlers[funOptions.handlerPath.split('/')[1]][funOptions.handlerName]}
+}
+
+module.exports = class OffLineBuilder {
+    constructor(serviceBuilder, options) {
+        this.serviceBuilder = serviceBuilder || new ServiceBuilder();
+        this.handlers = {};
+
+        //Avoid already wrapped exception when offline is instanciated many times
+        //Problem if test are instanciated serveral times
+        //FIXME, we could refactor index to have an handlerFactory and just instanciate offline with a factory test stub
+        if (functionHelper.createHandler.restore) {
+            functionHelper.createHandler.restore();
+        }
+        this.options = options || {};
+    }
+
+    addFunctionConfig(functionName, functionConfig, handler) {
+        this.serviceBuilder.addFunction(functionName, functionConfig);
+        const funOptions = functionHelper.getFunctionOptions(functionConfig, functionName, '.');
+        const handlerPath = funOptions.handlerPath.split('/')[1];
+        this.handlers[handlerPath] = this.getFunctionIndex(funOptions.handlerName, handler);
```

Comment:
Given that you defined getFunctionIndex as a static function, you should call it using this.constructor or the class name. Chose the method that suits better to your needs (I'd go with this.constructor).